examples/nextjs-app-dir-rate-limit/middleware.ts

```diff
@@ -18,4 +18,4 @@ const aj = arcjet({
   ],
 });
 
-export default createMiddleware(aj);
\ No newline at end of file
+export default createMiddleware(aj);
```

Comment:
How can we use this if we already have some other middleware e.g. Clerk/Auth.js & Arcjet? How can they be chained?

Reply:
Like any other middleware. Next.js middleware is nowhere near as nice as Express middleware or SvelteKit hooks. I've been thinking about a potentially cool package for dealing with Next.js middleware but that is not the responsibility of this package.